.github/workflows/apiBreakTest.yml

```diff
@@ -0,0 +1,216 @@
+name: Public Interface Breakage Detection
```

Comment:
We've been using `snake_case` for workflow file names. 
Could you rename this file to `api-breaking-changes-detection.yml` for consistency?